### PR TITLE
[Feature] Add template and voter login component

### PR DESCRIPTION
**Pull request by usmanoa**
### What does this PR do?
- This PR adds a default template and voter login form

#### Description of Task to be completed?
- Add reusable template and voter login component

#### Checklist:
- [x] Add default template
- [x] Add voter login component
- [x] Add voter input validation

#### trello link
https://trello.com/c/MKNnj7im

### CODE REVIEW

**Review by xand6r — APPROVED** [review]
Looks good, I appreciate your Javascript comments. Please try to do the same for the html so  we can add functions to it without reading it too much. Cheers

**Comment by Way2nnadi** [comment]
@usmanoa travis is failing, make sure to look at the what is causing the failure and fix it

File: src/components/login/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+function Login() {
+    const [nin, setNin] = useState('');
+    const [phoneNumber, setPhoneNumber] = useState('');
+
+/**
+ * Handles users NIN number
+ * @function
+ * @param {event} event - the event of 
+ */
+    const handleChangeNin = ({ target }) => {
```
Comment (Way2nnadi) [inline src/components/login/components/Login.js:24]: fix the indent here

File: src/components/login/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+            setNin(target.value);
+        }
+    };
+
+/**
+ * Handles users phone number
+ * @function
+ * @param {event} event - the event of 
+ */
+    const handleChangePhoneNumber = ({ target }) => {
```
Comment (Way2nnadi) [inline src/components/login/components/Login.js:39]: fix indent

**Review by Onyimatics — CHANGES_REQUESTED** [review]
Nice work @usmanoa. However, our build is failing, please try and fix it. Thank you.

File: src/components/login/components/Login.js (hunk 2 above)
Comment (xand6r) [reply src/components/login/components/Login.js:39]: Thank you uche

File: src/components/login/components/Login.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+    const handleSubmit = event => {
+        event.preventDefault();
+        console.log({ nin, phoneNumber });
+    };
+
+    return (
+        <form className="form" onSubmit={handleSubmit}>
+            <div className="form_subSection">
+                <Title className="form__heading" level={3}>Sign in to your Account</Title>
```
Comment (Way2nnadi) [inline src/components/login/components/Login.js]: save string in `constants.js` file

Comment (usmanoa) [reply src/components/login/components/Login.js]: Thanks, I'll do that

File: src/components/login/components/Login.js (hunk 1 above)
Comment (usmanoa) [reply src/components/login/components/Login.js:24]: Thanks, I'll fix it.

**Comment by Onyimatics** [comment]
[Users should be able to login to voteright](https://trello.com/c/MKNnj7im/41-users-should-be-able-to-login-to-voteright)

**Review by Onyimatics — APPROVED** [review]
Well done @usmanoa  I can see the build is passing now.

**Pull request merged**